db/migrate/20241122143740_create_account_recovery_requests.rb

```diff
@@ -0,0 +1,11 @@
+class CreateAccountRecoveryRequests < ActiveRecord::Migration[8.0]
+  def change
+    create_table :account_recovery_requests do |t|
+      t.integer :code, null: false, index: { unique: true }
```

Comment:
Need to encrypt this on the production PR

Reply:
Also, the AccountRecoveryRequest table should have multiple codes attached to it as we want to allow the user to use multiple codes, all with a arbitrary expiration time

Reply:
(Ha, I should have read the whole thing before I said you should encrypt the code...)